docs/source/examples/github.rst

```diff
@@ -176,3 +176,36 @@ This is the same as::
     me = g.me() # or me = g.user(your_user_name)
     if me.update(new_name, blog, company, bio=bio):
         print('Profile updated.')
+
+
+Logging in as an Application, or Application Installation
+---------------------------------------------------------
+
+To login as an application you only need the app_id and private key.
+
+::
+
+    # Read the key file created by GitHub
+    with open('path/to/private-key.pem', 'rb') as pem_file:
+        key_file_pem = pem_file.read()
+
+    # start with anonymous GitHub object
+    gh = github3.GitHub()
+
+    # login as application, and access installations
+    gh.login_as_app(private_key_pem=key_file_pem, app_id=app_identifier)
+    gh.app_installations()
+
+    # OR obtain installation token for additional App APIs.
+    # Installation ID is provided in GitHub webhook during a user's install.
+    gh.login_as_app_installation(key_file_pem, app_identifier, install_id)
+    access_token = gh.session.auth.token
+    # (i.e. list all repos accessible to this installation)
+    response = requests.get('https://api.github.com/installation/repositories',
```

Comment:
Why would users do this as opposed to using the library?

Reply:
I think this API is not yet implemented.

@eddiewebb For a lot of the APIs you actually need app auth. With app installation auth you mainly have access to the resources of that installation (e.g. a repo or orga).